### DISCUSSION TITLE
Integrity Check failed after Update to V6 and fixing of Calendar update issue

### DISCUSSION THREAD

**Discussion by GUNT0815** [nextcloud/all-in-one#2742] [Questions] [answered]
### Steps to reproduce
1. Update nextcloud-aio to v6
2. Calenar update -> failed
3. Manual workaround as discussed here

### Expected behavior 

file integrity has no negative status

### Actual behavior 

Technical information
=====================
The following list covers which files have failed the integrity check. Please read
the previous linked documentation to learn more about the errors and how to fix
them.

Results
=======
- core
	- INVALID_HASH
		- core/js/mimetypelist.js

Raw output
==========
Array
(
    [core] => Array
        (
            [INVALID_HASH] => Array
                (
                    [core/js/mimetypelist.js] => Array
                        (
                            [expected] => 4b0a99fbab7bc4824176101150bd0ab2b553e128a390e0f872580db4541462b31aad13099fd41533b6695d1b185dabca26a45e9a78227cfb9479eafa439c7bff
                            [current] => a8e818876a7eff2a2137601eafc6c7d96b99cbf27d94ff9bb59eb237bdab92076ecc2828242bb37551c9c8b6772c0718887bd49cee5b49b3be48d3b9ecaa558f
                        )

                )

        )

)

#### Nextcloud AIO version 
6.0.0

**Comment by szaimen** [comment, marked as answer]
Hi, this is nothing to worry about: you can just ignore it.

It usually should get fixed automatically with the next Nextcloud update which is planned for beginning of July.
